src/openlcb/MemoryConfigClient.hxx

```diff
@@ -135,9 +220,19 @@ public:
     {
     }
 
+    enum ResultCodes
+    {
+        // Internal error codes generated by the send flow
+        OPERATION_PENDING =
+            DatagramClient::OPERATION_PENDING, //< cleared when done is called.
+        TIMEOUT = Defs::ERROR_PERMANENT |
+            Defs::OPENMRN_TIMEOUT, //< Timeout waiting for ack/nack.
+    };
+
 private:
     Action entry() override
     {
+        request()->resultCode = OPERATION_PENDING;
         switch (request()->cmd)
         {
         case MemoryConfigClientRequest::CMD_READ:
```

Comment:
Not a big deal, but the style guide says that these case statements should be indented.

Reply:
done